packages/data-explorer-ui/src/components/Export/components/ExportToTerra/exportToTerra.tsx

```diff
@@ -37,7 +36,7 @@ export const ExportToTerra = ({
       {/* Export is successful */}
       {isSuccess && (
         <ExportToTerraReady
-          terraUrl={buildExportToTerraUrl(
+          exportURL={buildExportToTerraUrl(
```

Comment:
For possible re-use of the new `ExportToTerraReady` component I have re-named `terraUrl` to `exportURL` - the old `ExportToTerraReady` component now [references props](https://github.com/clevercanary/data-explorer/pull/225/files#diff-bc2ab4d459ad32a82c208249f12e7aead0f06f25ce9a86e372c22c15fb33e8e8R17) from the new component.